### PR TITLE
Merge Rhythm-ui mk-2 into the default master branch

### PR DESCRIPTION
**Pull request by christopher-hopper**
## Proposed Change

I propose that it is time to move to a standard branching model for Rhythm UI. This could start by merging the mk-2 branch into the default `master` branch. 

## Motivation

In order to more easily add, configure, manage and execute build pipelines, we need to bring the  default branch (currently set  as `master`) up-to-date with the latest development branch. Until this happens many features of build pipelines, including build status badges, some pipeline triggers for mirroring, and more will not be available or more difficult to implement.

My understanding from @jennasalau was that the mk-2 branch is not the default in an attempt to hide the HEAD commits from the public. While that is admirable, its not effective. Anyone familiar with Git and Github has no trouble identifying that the current HEAD branch is actually this mk-2 branch.

In short, not having the current work in the default branch is more of a hindrance than a help.

### CODE REVIEW

**Comment by jennasalau** [comment]
I think this is fine to do.

We will have to come back to an old commit to port over all the forms code that exists here into the new web components pattern.

The last release was here:
https://github.com/DeloitteDigitalAPAC/rhythm-ui/tree/v0.1.9
